project/gradle.properties

```diff
@@ -17,4 +17,5 @@ org.gradle.jvmargs=-Xmx1536m
 # org.gradle.parallel=true
 
 android.useAndroidX=true
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=true
+android.jetifier.blacklist=.*datadog.*
```

Comment:
is this still necessary?

Reply:
Yes. the version used is 9.1.0 in sample app